backend/src/schema/resolvers/searches.js

```diff
@@ -0,0 +1,69 @@
+const transformReturnType = record => {
+  return {
+    __typename: record.get('type'),
+    ...record.get('resource').properties,
+  }
+}
+export default {
+  Query: {
+    findResources: async (_parent, args, context, _resolveInfo) => {
+      const { query, limit } = args
+      const { id: thisUserId } = context.user
+      // see http://lucene.apache.org/core/8_3_1/queryparser/org/apache/lucene/queryparser/classic/package-summary.html#package.description
+      const myQuery = query + '*'
```

Comment:
I am not sure about this. It is not meant to search full phrases. If you enter more then one word, than only the last word is searched with the wildcard. The other words must be written completely to be found. I am not sure if this is the purpose of a search.

Reply:
we can add it back @Mogge , but it was kinda broken for me... so if I search for `not blocked` in the cypress test it would not find any posts with the content `blocked`, only if I search for `blocked` would it work.
I think maybe the only reason it started working as I expected is that with adding just the wildcard at the end of the phrase `not blocked` means that it is only looking for `blocked`, but at least it actually finds results.

Reply:
I have tested this out with `ökologischer Garten` in the Post content and searching for `öko garten` as you suggested in Discord @Mogge... I can see what you are concerned about.
I have added this line of code back :point_up_2:,  but now it is failing the cypress tests when searching for `not blocked` with the content being `cause I'm not blocked` ... what would you suggest?